### PR TITLE
gui frontend for projects

### PR DESCRIPTION
**Pull request by abacz**

### CODE REVIEW

**Comment by agpul** [comment]
Do we plan to do something with this view? 
image

I mean USAGE section. For me, it should look better.

**Comment by agpul** [comment]
> Do we plan to do something with this view?
> image
> 
> I mean USAGE section. For me, it should look better.

For eg. "rt" is closer to the Customer country than to Access reason. Moreover, the distances are too long for me

**Comment by abacz** [comment]
Yes @agpul, @jarekzet will fix it.

**Comment by jarekzet** [comment]
Hi, made some fixes on order configuration list
Opera Zdjęcie_2019-07-05_112737_localhost

File: app/views/services/configurations/show_normal.html.haml [hunk 1]
```diff
@@ -11,40 +11,50 @@
...
+            "data-action" => "change->project-item#projectChanged",
+            "data-target" => "project-item.project"
+          .input-group-append
+            = link_to "Add new project", new_project_path,
+              class: "btn btn-primary", remote: true
+
+    %h3 Usage
+    %dl.mb-4
+      %dt Customer typology
+      %dd= t "simple_form.options.defaults.customer_typology.#{@project_item.project.customer_typology}"
```
Comment (bwilk) [inline app/views/services/configurations/show_normal.html.haml]: N + 1 query here

Comment (bwilk) [reply app/views/services/configurations/show_normal.html.haml]: Create factories in specs for project_item.project.customer_typology to fix `./spec/features/order_spec.rb:364 ` and other tests

**Pull request merged**